src/main/java/org/utplsql/cli/LoggerConfiguration.java

```diff
@@ -0,0 +1,64 @@
+package org.utplsql.cli;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.ConsoleAppender;
+import com.zaxxer.hikari.HikariDataSource;
+import org.slf4j.LoggerFactory;
+import org.utplsql.api.TestRunner;
+
+class LoggerConfiguration {
+
```

Comment:
if class is non-instantiable pleaso provide private constructor with UnsupportedOperationException
```suggestion
    private LoggerConfiguration() {
       throw new UnsupportedOperationException();
    }
```